db/schema.rb

```diff
@@ -584,6 +584,8 @@
     t.jsonb "auto_assignment_config", default: {}
     t.boolean "lock_to_single_conversation", default: false, null: false
     t.bigint "portal_id"
+    t.string "business_name"
+    t.boolean "custom_sender_name_enabled", default: true, null: false
```

Comment:
@sojan-official @tejaswinichile I would suggest to keep this an enum rather than a boolean. Something like sender_name_type as friendly:0 or professional:1

Reply:
Okay. 👍🏼